data/secscan_model/datatypes.py

```diff
@@ -95,7 +104,13 @@ def to_dict(self):
                                 "FixedBy": v.FixedBy,
                                 "Description": v.Description,
                                 "Name": v.Name,
-                                "Metadata": v.Metadata,
+                                "Metadata": {
```

Comment:
Yes again, why translate the name from the original VulnerabilityReport into something new? If you have the ability to keep the field names consistent across both? Is this not possible for some reason?

Reply:
This is to avoid changes in container security operator as suggested by @alecmerdler. CSO populates the `Metadata` as is into the custom resource.